app/components/message_attachments/attachment_image/attachment_image.js

```diff
@@ -48,6 +48,16 @@ export default class AttachmentImage extends PureComponent {
         }
     }
 
+    componentWillReceiveProps(nextProps) {
```

Comment:
change this to componentDidUpdate and calling ImageCacheManager should be enough